src/main/resources/mappers/lt/velykis/roberta/budget/app/account/account.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE mapper PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN"
+        "http://mybatis.org/dtd/mybatis-3-mapper.dtd">
+
+<mapper namespace="lt.velykis.roberta.budget.app.account.AccountRepository">
+    <!--    result maps     -->
+    <resultMap id="accountMap" type="lt.velykis.roberta.budget.app.account.Account">
+        <id column="id" property="name"/>
+        <result column="id" property="id"/>
+        <result column="name" property="name"/>
```

Comment:
are the `<result..>` mappings necessary? Wouldn't myBatis map them automatically?

Reply:
Deleted resultMap and added resultType instead